src/pages/homeView.js

```diff
@@ -4,7 +4,7 @@ import Button from "../components/global/Button";
 function homeView() {
   return (
     <div>
-      <Button icon="rightArrow" text="Dashboard" path="/home" />
+      <Button icon="rightArrow" text="Dashboard" path="/home" handleOnClick={() => {console.log('asda')}}/>
```

Comment:
Should remove this since its going into main

Reply:
good spot!